docs/Reference/README.md

```diff
@@ -0,0 +1,53 @@
+# Reference Documentation
+
+How to use the Cosmos Cash Reference Documentation.
+
+[TOC]
+
+
+
+## Context
+
+In Cosmos Cash we use the  *Grand Unified Theory of Documentation* (David Laing) as described by [Divio](https://documentation.divio.com/) as a basis for our documentation strategy.
+
+This approach outlines four specific use cases for documentation:
+
+* [Tutorials](../Tutorials/README.md)
+* [How-Tos](../How-To/README.md)
+* [Explanation](./Explanation/README.md)
+* [Reference](./README.md)
+
+This section addresses the last of these: **Reference documentation**. 
+
+For further background please see [the ADR relating to the documentation structure](./ADR/adr-002-docs-structure.md). 
```

Comment:
see comment line 22 in docs/Explanation/README.md

hmmm I see a pattern here, do we really need to repeat the grand theory and core basics for each content type? wdyt about moving out and having a single instance of this information instead of repeating in each content type README?